src/main/java/com/stripe/model/Topup.java

```diff
@@ -0,0 +1,199 @@
+package com.stripe.model;
+
+import com.stripe.exception.*;
+import com.stripe.net.APIResource;
+import com.stripe.net.RequestOptions;
+
+import java.util.Map;
+
+public class Topup extends APIResource implements MetadataStore<Topup>, HasId {
+    String id;
+    String object;
+    Long created;
+    Boolean livemode;
+    Integer amount;
+    String currency;
+    Map<String, String> metadata;
+    Source source;
+    String status;
+    String balanceTransaction;
```

Comment:
Hmm, do you know how this works if a user requests to expand the `BalanceTransaction`?

https://github.com/stripe/stripe-java/blob/master/src/main/java/com/stripe/model/Transfer.java#L20?

Reply:
Good catch, I didn't know that `BalanceTransaction` could be expanded. Fixed in 18f6b86